src/basic/process-util.c

```diff
@@ -175,7 +175,7 @@ int get_process_cmdline(pid_t pid, size_t max_columns, ProcessCmdlineFlags flags
 
         bool eight_bit = (flags & PROCESS_CMDLINE_USE_LOCALE) && !is_locale_utf8();
 
-        ans = escape_non_printable_full(t, max_columns, eight_bit);
+        ans = escape_non_printable_full(t, max_columns, eight_bit, false);
```

Comment:
maybe add `/* force_ellispsis= */` before the `false`

(or better: turn this into a flag)